CHANGES.txt

```diff
@@ -4,6 +4,8 @@ Changes for Crate Admin Interface
 
 Unreleased
 ==========
+  
+ - Fix : Reduced the number of logs.
```

Comment:
just write: Reduced the amount of log output to brower console when issuing SQL statements.